pkg/grpc/server.go

```diff
@@ -289,87 +264,25 @@ func (s *server) getGRPCServer() (*grpc.Server, error) {
 		opts = append(opts, grpc.KeepaliveParams(keepalive.ServerParameters{MaxConnectionAge: *s.maxConnectionAge}))
 	}
 
-	if s.authenticator != nil {
-		err := s.generateWorkloadCert()
-		if err != nil {
-			return nil, err
-		}
-
-		//nolint:gosec
-		tlsConfig := tls.Config{
-			ClientCAs:  s.signedCert.TrustChain,
-			ClientAuth: tls.RequireAndVerifyClientCert,
-			GetCertificate: func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
-				return &s.tlsCert, nil
-			},
-		}
-
-		// In the internal server, enforce minimum version TLS 1.2
```

Comment:
Looks like this is not enforced anymore in the new code? We need to make sure we are enforcing TLS 1.2+

Reply:
It is in the spiffe library https://github.com/spiffe/go-spiffe/blob/3000ff09ead100134b2828fec1151b0419a988b0/v2/spiffetls/tlsconfig/config.go#L239